packages/experimental/Divider/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@fluentui-react-native/divider",
+  "version": "0.1.0",
+  "description": "A cross-platform Divider component using the Fluent Design System",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "clean": "fluentui-scripts clean",
+    "depcheck": "fluentui-scripts depcheck",
+    "just": "fluentui-scripts",
+    "lint": "fluentui-scripts eslint",
+    "test": "fluentui-scripts jest",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "prettier": "fluentui-scripts prettier",
+    "prettier-fix": "fluentui-scripts prettier --fix true"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui-react-native.git",
+    "directory": "packages/experimental/Divider"
+  },
+  "dependencies": {
+    "@fluentui-react-native/framework": "0.8.33",
+    "@fluentui-react-native/icon": "^0.17.3",
+    "@fluentui-react-native/text": ">=0.19.11 <1.0.0",
+    "@fluentui-react-native/tokens": ">=0.20.3 <1.0.0",
+    "@fluentui-react-native/theme-tokens": "^0.23.2",
+    "@fluentui-react-native/use-styling": ">=0.9.1 <1.0.0"
+  },
+  "devDependencies": {
+    "@fluentui-react-native/eslint-config-rules": "^0.1.1",
+    "@fluentui-react-native/test-tools": ">=0.1.1 <1.0.0",
+    "@fluentui-react-native/scripts": "^0.1.1",
+    "react": "17.0.2",
+    "react-native": "^0.68.0"
+  },
+  "peerDependencies": {
+    "react": "17.0.2",
+    "react-native": "^0.68.0"
+  },
```

Comment:
I think there should be a section for `rnx-kit` here, similar to every other package. See #2564 for examples.